### ISSUE TITLE
GitHub Flavoured UI

### ISSUE THREAD

**Issue by omegaui** [vivekweb2013/gitnoter#50]
The Application will look more impactful if the UI elements follow GitHub's Styling.

**Comment by vivekweb2013** [comment]
Agree. Thanks for creating the issue. Would you like to work on this? You can simply run the frontend react application by referring to the instructions mentioned in this [readme file](https://github.com/vivekweb2013/gitnoter/blob/main/frontend/README.md).

**Comment by omegaui** [comment]
> Agree. Thanks for creating the issue. Would you like to work on this? You can simply run the frontend react application by referring to the instructions mentioned in this [readme file](https://github.com/vivekweb2013/gitnoter/blob/main/frontend/README.md).

If this can be achieved just by editing the CSS files then, I'll love to implement it.

**Comment by vivekweb2013** [comment]
yes. you can create and include the scss files or if you want you can also add stylings inline to the component. I'm using [MUI components](https://mui.com/material-ui/getting-started/usage/) and it provides several ways to style a component.

**Comment by cool-pants** [comment]
Is someone working on this? If not, I would love to pick it up..

**Comment by omegaui** [comment]
@cool-pants 

Go Ahead for sure!

Currently my college Unit Test are about to begin,
So I won't be able to start until the next Saturday.

In short,
Give it a hit.

**Comment by Senor-Ducky** [comment]
Hey Vivek, do you mind if i start working on this?

**Comment by vivekweb2013** [comment]
> Hey Vivek, do you mind if i start working on this?

yeah sure. @cool-pants are you working on this? if not I think @Senor-Ducky can start working on this

**Comment by andrei-io** [comment]
Hello! I suggest using the already made CSS library made by Github @cool-pants @Senor-Ducky @vivekweb2013 
https://github.com/primer/css

**Comment by Senor-Ducky** [comment]
> Hello! I suggest using the already made CSS library made by Github @cool-pants @Senor-Ducky @vivekweb2013 https://github.com/primer/css

sure thing!

**Comment by cool-pants** [comment]
> > Hey Vivek, do you mind if i start working on this?
> 
> yeah sure. @cool-pants are you working on this? if not I think @Senor-Ducky can start working on this

Well i have started the work, and done most of it. @Senor-Ducky sorry for the late reply.

**Comment by vivekweb2013** [comment]
@cool-pants okay please continue.
Hey @Senor-Ducky. Could you please pick [issue 49](https://github.com/vivekweb2013/gitnoter/issues/49) instead.
